tailwind.config.js

```diff
@@ -15,10 +15,24 @@ module.exports = {
         screens: {
             md: { min: '800px' }
         },
-        extend: {},
         colors: {
             transparent: 'transparent',
             current: 'currentColor',
+            currentColor: '#ED6C5A',
+            orange: {
```

Comment:
I added the orange that was in the mockup from Kei. We can change this to our future colors. 
Set it up so we can change it easily. We could even allow theming to different colors.